src/features/applicationMetadata/index.d.ts

```diff
@@ -1,5 +1,10 @@
 import type { IDataType, ITitle } from 'src/types/shared';
 
+type ILogoOptions = {
+  source: 'org' | 'resource';
+  displayAppOwnerNameInHeader: boolean;
```

Comment:
Isn't both of these optional? 🤔 According to the schema they both are, so we should probably make sure to check that they're set. I would assume `displayAppOwnerNameInHeader` defaults to false.

Reply:
Ah yes they are. And yes, `displayAppOwnerNameInHeader` defaults to `false`

Reply:
On second thought. `displayAppOwnerNameInHeader` defaults to `false` in `app-lib-dotnet`. That means that it will always be set in the frontend, so long as the `logo` property is defined in `applicationmetadata.json`. `source` is optional, though.

Reply:
Ended up making `displayAppOwnerInHeader` to make sure we are backwards compatible with older versions of Altinn.Core